actions/release/create/entrypoint/main_test.go

```diff
@@ -35,8 +35,10 @@ func TestEntrypoint(t *testing.T) {
 			Expect     = NewWithT(t).Expect
 			Eventually = NewWithT(t).Eventually
 
-			api      *httptest.Server
-			requests []*http.Request
+			api                 *httptest.Server
```

Comment:
I see that we changed the `main_test.go` for the `release/create` action, but I don't see any changes to the `main.go` file for the `release/create` action itself.

Am I missing something?

Reply:
changed the wrong test file, will revert this part.